olp-cpp-sdk-core/include/olp/core/http/NetworkRequest.h

```diff
@@ -37,8 +37,6 @@ class CORE_API NetworkRequest final {
  public:
   /// Short type alias for HTTP request body.
   using RequestBodyType = std::shared_ptr<const std::vector<std::uint8_t>>;
-  /// Short type alias for HTTP request header.
-  using RequestHeadersType = std::vector<std::pair<std::string, std::string>>;
```

Comment:
This is an API break. Please avoid changing public defines.
Better just provide an additional define in HttpResponse.

Reply:
I mean we should keep it but use the HeadersType as base:

```cpp
using RequestHeadersType = HeadersType;
```

and then leave the rest of the class untouched.